src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,21 @@
-export const CommentList = () => (
-  <>Put the list here</>
-);
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ posts, users, comments }) => {
```

Comment:
```suggestion
export const CommentList = ({ comments }) => {
```
pass here only comments